### PR TITLE
Add Getscreen.me

### PR DESCRIPTION
**Pull request by faridshamsutdinov**

### CODE REVIEW

File: entries/g/getscreen.me.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Getscreen.me": {
+    "domain": "getscreen.me",
+    "tfa": [
+      "email",
+      "custom-software"
```
Comment (phallobst) [inline entries/g/getscreen.me.json]:
```suggestion
      "custom-software",
      "totp"
```

File: entries/g/getscreen.me.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    ],
```
Comment (phallobst) [inline entries/g/getscreen.me.json:11]:
```suggestion
    ],
    "custom-software": [
      "Telegram"
    ],
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: img/g/getscreen.me.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg width="32" height="32" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)"><path fill-rule="evenodd" clip-rule="evenodd" d="M25.905-.117A6.095 6.095 0 0132 5.978v19.81a6.095 6.095 0 01-6.095 6.095H6.095A6.095 6.095 0 010 25.788V5.978A6.095 6.095 0 016.095-.117h19.81z" fill="#448AFF"/><mask id="a" maskUnits="userSpaceOnUse" x="0" y="-1" width="32" height="33"><path fill-rule="evenodd" clip-rule="evenodd" d="M25.905-.117A6.095 6.095 0 0132 5.978v19.81a6.095 6.095 0 01-6.095 6.095H6.095A6.095 6.095 0 010 25.788V5.978A6.095 6.095 0 016.095-.117h19.81z" fill="#fff"/></mask><g mask="url(#a)"><path fill-rule="evenodd" clip-rule="evenodd" d="M9.642 8.973c-.911-.013-1.377 1.09-.732 1.734l3.44 3.44c.396.396.396 1.04 0 1.436L-.438 28.369l3.951 3.95L16.3 19.535a1.016 1.016 0 011.437 0l3.439 3.439c.645.645 1.747.18 1.734-.732l-.167-12.1a1.016 1.016 0 00-1.002-1.001L9.642 8.973z" fill="#fff"/></g></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h32v32H0z"/></clipPath></defs></svg>
```
Comment (Carlgo11) [inline img/g/getscreen.me.svg]: I cleaned up the code a bit and have two different versions for you to choose from:
1. The original (but cleaned up) icon: __(865 Bytes)__

2. An outline of the logo: __(594 Bytes)__

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/g/getscreen.me.svg (hunk 3 above)
Comment (faridshamsutdinov) [reply img/g/getscreen.me.svg]: @Carlgo11 2

Comment (Carlgo11) [reply img/g/getscreen.me.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path fill="#448aff" d="M 6.0957 0 A 6.095 6.095 0 0 0 0 6.0957 L 0 25.904 A 6.095 6.095 0 0 0 0.31641 27.732 L 12.35 15.699 C 12.746 15.303 12.746 14.66 12.35 14.264 L 8.9102 10.824 C 8.2652 10.18 8.7316 9.0768 9.6426 9.0898 L 21.74 9.2578 A 1.016 1.016 0 0 1 22.742 10.26 L 22.91 22.359 C 22.923 23.271 21.821 23.737 21.176 23.092 L 17.736 19.652 A 1.016 1.016 0 0 0 16.301 19.652 L 4.2676 31.682 A 6.095 6.095 0 0 0 6.0957 32 L 25.904 32 A 6.095 6.095 0 0 0 32 25.904 L 32 6.0957 A 6.095 6.095 0 0 0 25.904 0 L 6.0957 0 z"/></svg>
```

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**